generators/server/templates/src/main/resources/templates/error.html.ejs

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="http://www.thymeleaf.org ">
+      xsi:schemaLocation="http://www.thymeleaf.org" th:lang="${#locale.language}">
```

Comment:
are we pointing to user selected language?

Reply:
No it seems to be always english/the default language at least it doesn't make any difference which language is selected or saved in the profile/